src/connectors/systemService.ts

```diff
@@ -214,7 +214,7 @@ export class SystemService extends BaseService {
    * Find the url of an asset by a given id.
    */
   findAssetUrl = async (id: string): Promise<string | null> => {
-    const result = await this.baseFindById(id, 'asset')
+    const result = await this.knexRO.from('asset').where({ id }).first()
```

Comment:
Would this affect by the replication lag? Like uploading images to user avatar or draft editor.

Reply:
oh, yes, it would be affect in these cases. I will revert this change